src/models/user.ts

```diff
@@ -168,6 +169,11 @@ export class User extends SubscribableModel {
     return typeof this._userData.oauthServices !== 'undefined' ? this._userData.oauthServices.map(oauthService => new OAuthService(oauthService, this.sessionHandler)) : undefined
   }
 
+  async authorize (params: { clientIdentifier: string, redirectUrl: string, responseType: oAuthResponseTypes, state: string }) {
```

Comment:
This name needs to be more explicit. `authorize` is generic and doesn't explain what this method is used for.